### DISCUSSION TITLE
Adding Multiple Cars in Home Assistant

### DISCUSSION THREAD

**Discussion by bcre3306** [teslamate-org/teslamate#4423] [Getting started] [answered]
Hi

I recently got teslamate up and running and integrated into HA but I am having an issue getting the second car to show up in HA.  So I have two cars in the same Tesla account, they both show up in teslamate, both flow over to MQTT(Car 1 & 2) but in HA I only see car 1.  I'm sure this has to do with my mqtt_sensors.yaml as this only addresses car 1 but I'm unsure how to add the second.  Should I add another yaml file with the second car and add that to the configuration.yaml?

Thank you

**Comment by vincep5** [comment, marked as answer]
Yes, you can use the same file.  Take the example of mqtt_sensors.yaml and copy it over to notepad.  Do a find and replace so that it replaces a few things
- teslamate/cars/1/ to teslamate/cars/2/
- the unique id teslamate_1 to teslamate_2
- the device: sections should match properly
I would make a top level comment for each vehicle so that you can see each section easier.

What I did was make the object_id and unique_object_id contain the name of my named vehicles so that it was clear as to which item was for which vehicle.  All of it is placed in the same yaml

**Reply by bcre3306** [reply]
Figured it out while you sent this.  One thing you missed was the teslamate_active sections needed a unique name also so I changed them to teslamate_active2.

Thank you!!
